transaction-status/src/lib.rs

```diff
@@ -976,6 +1091,13 @@ pub struct UiRawMessage {
     pub address_table_lookups: Option<Vec<UiAddressTableLookup>>,
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct UiAccountsList {
+    pub signatures: Vec<String>,
+    pub account_keys: Vec<ParsedAccount>,
```

Comment:
What do you think about nesting this under a `message` field to keep its response shape closer to the other details variants?

Reply:
I suppose that could make switching over to this response type slightly easier for users that are processing `message.accountKeys.` But it would add something like 10 bytes per transaction to the response. My leaning is not worth the extra payload. How strongly do you feel about it?

Reply:
Yeah, probably not worth it. I don't feel strongly